src/utils.js

```diff
@@ -306,7 +306,7 @@ const utils = {
 	 */
 	match(text, pattern) {
 		// Simple patterns
-		if (pattern.indexOf("?") == -1) {
+		if (pattern.indexOf("?") == -1 && pattern.indexOf("|") == -1) {
```

Comment:
You can't modify the logic of `match` method in utils because it is used in many places in the core and official modules, to match service names, action names, event names...etc. So this change affects other search logics as well and it can cause major breaking changes, too.

Reply:
Oh I see, I thought it was only used by hooks. You wouldn't want to allow users to add "a|b" as an action, service or event name, which would have undefined behavior...

But just to know, do you think the feature proposal by itself is a good addition to the framework?

Reply:
Yeah, this change can be a good feature, but the pipe sign should be parsed inside the ActionHooks middleware only.

Reply:
Great! I'll get to it sometimes this week.